tests/main/listing/task.yaml

```diff
@@ -13,6 +13,9 @@ execute: |
     if [ "$SPREAD_BACKEND" = "linode" -o "$SPREAD_BACKEND" == "qemu" ] && [ "$SPREAD_SYSTEM" = "ubuntu-core-16-64" ]; then
         echo "With customized images the ubuntu-core snap is sideloaded"
         expected='^core .* [0-9]{2}-[0-9.]+(~[a-z0-9]+)?(\+git[0-9]+\.[0-9a-f]+)? +x[0-9]+ +core *$'
+    elif [ "$SRU_VALIDATION" = "1" ]; then
+        echo "When sru validation is the ubuntu-core snap is installed from the store"
+        expected='^core .* [0-9]{2}-[0-9.]+(~[a-z0-9]+)?(\+[0-9]+\.[0-9a-f]+)? +[0-9]+ +canonical +core *$'
```

Comment:
from the comment I was expecting this to be `^ubuntu-core`.

Reply:
@chipaca nice comment, you are right